index.md

```diff
@@ -16,6 +29,54 @@ This session aims to introduce the use of Docker containers with the goal of usi
 > If you are looking for a lesson on using *singularity* containers (instead of Docker), see this lesson:
 > * [Reproducible Computational Environments Using Containers: Introduction to Singularity](https://carpentries-incubator.github.io/singularity-introduction/)
 
+## Learner profiles
+
+To help give an idea of the target audience for this lesson we have included 
+some example learner profiles.
+
+*Nelson is a graduate student in microbiology.* They have experience in running unix shell
+commands and using libraries in R for the bioinformatics workflows they have devleoped.
+They are expanding their analysis to run on 3000 genomes in 200 samples and they have
+started to use the local cluster to run their workflows.  The local reserach computaing
+facilitator has advised them that Docker could be useful for running their workflows on
+the cluster. They'd like to make use of existing containers that other bioinformaticians
+have made so they want to learn how to use Docker. They would also be interested in
+creating their own Docker images for other lab members and collaborators to re-use their
+workflow.
+
+*Caitlin is a second year undergraduate in computer science examining Docker for the first
+time.* She has heard about Docker but does not really know what it achieves or why it is
+useful. She is reasonably confident in using the Unix shell, having used it briefly in
+her first year papers. She is keen to find jump-off points to learn more about technical
+details and alternative technologies that are also popular, having heard that container
+technologies are widely used within industry.
+
+*Xu, a materials science researcher, wants to package their software for release with
+a paper to help ensure reproducibility.* She has written some code that makes use of a
+series of Python libraries to undertake analysis of a compound. He wants to (or is
+required to) make her software available as part of the paper submission. She
+understands why Docker is important in helping to ensure reproducibility but not the
+process and low-level detail of preparing a container and archiving it to obtain a DOI
+for inclusion with the paper submission.
+
+*Bronwyn is a PhD student running Python/R scripts on their local laptop/workstation.*
```

Comment:
also mix of pronouns in this one between the summary sentence here and rest of paragraph